examples/segmentation/train_deeplabv3_ade20k.py

```diff
@@ -0,0 +1,359 @@
+parser.add_argument('--eval_batch_size', help='Validation dataloader per-device batch size', type=int, default=128)
+
+# Model command-line arguments
+parser.add_argument('--backbone_arch',
+                    help='Architecture to use for the backbone.',
+                    default='resnet101',
+                    choices=['resnet50', 'resnet101'])
+parser.add_argument('--sync_bn',
+                    help='Use sync BatchNorm. Recommended if the per device microbatch size is below 16',
+                    action='store_true')
+parser.add_argument('--cross_entropy_weight', help='Weight to scale the cross entropy loss', type=float, default=0.375)
+parser.add_argument('--dice_weight', help='Weight to scale the dice loss', type=float, default=1.125)
+
+# Optimizer command-line arguments
+parser.add_argument('--learning_rate', help='Optimizer learning rate', type=float, default=0.08)
+parser.add_argument('--momentum', help='Optimizer momentum', type=float, default=0.9)
+parser.add_argument('--weight_decay', help='Optimizer weight decay', type=float, default=5.0e-5)
+
+# Save checkpoint command-line arguments
+parser.add_argument('--save_checkpoint_dir',
+                    help='Directory in which to save model checkpoints',
+                    type=str,
+                    default='checkpoints/{run_name}')
+parser.add_argument('--checkpoint_interval',
+                    help='Frequency to save checkpoints',
+                    type=Time.from_timestring,
+                    default='1ep')
+
+# Load checkpoint command-line arguments, assumes resuming from a previous training run (as opposed to fine-tuning)
+parser.add_argument('--load_checkpoint_path', help='Path to the checkpoint to load', type=str)
+
+# Recipes command-line argument
+parser.add_argument('--recipe_name',
+                    help='Algorithmic recipes to be applied to the trainer',
+                    choices=['mild', 'medium', 'hot'])
+
+# Logger command-line arguments
+# Note: Only Weights and Biases to minimize arguments. Other loggers can be used by adjusting the script
+parser.add_argument('--wandb_logger', help='Whether or not to log results to Weights and Biases', action='store_true')
+parser.add_argument('--wandb_entity', help='WandB entity name', type=str)
+parser.add_argument('--wandb_project', help='WandB project name', type=str)
+
+# Trainer arguments
+parser.add_argument('--grad_accum', help='Amount of gradient accumulation if running on CPU', type=int, default=1)
+parser.add_argument('--run_name', help='Name of the training run used for checkpointing and logging', type=str)
+parser.add_argument('--seed', help='Random seed', type=int, default=17)
+parser.add_argument('--max_duration',
+                    help='Duration to train specified as a Time string',
+                    type=Time.from_timestring,
+                    default='128ep')
+
+args = parser.parse_args()
+
+IMAGENET_CHANNEL_MEAN = (int(0.485 * 255), int(0.456 * 255), int(0.406 * 255))
```

Comment:
Should be ADE20k mean, std here right?

Reply:
We use the mean and standard deviation from ImageNet, I'm guessing I got this from MMseg and the rational was that it is the normalization used for the pretrained model?